specifyweb/frontend/js_src/lib/components/Statistics/utils.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import type { State } from 'typesafe-reducer';
+
+import { useAsyncState } from '../../hooks/useAsyncState';
+import { ajax } from '../../utils/ajax';
+import type { IR, RA, RR } from '../../utils/types';
+import { keysToLowerCase } from '../../utils/utils';
+import { formatNumber } from '../Atoms/Internationalization';
+import { serializeResource } from '../DataModel/helpers';
+import type { SerializedResource } from '../DataModel/helperTypes';
+import { schema } from '../DataModel/schema';
+import type { SpQueryField, Tables } from '../DataModel/types';
+import { makeQueryField } from '../QueryBuilder/fromTree';
+import { addMissingFields } from '../DataModel/addMissingFields';
+
+/** Fetch statistics from the QueryBuilderAPI. */
+export function useStatAjaxHelper(
```

Comment:
`useStatAjaxHelper` is not a very clear name - I don't know what this helper does.

A better name might be `useFrontEndStat`